addons/centered/src/ember.js

```diff
@@ -0,0 +1,23 @@
+import { document } from 'global';
+import styles from './styles';
+
+function stylesToString(_styles) {
+  return Object.keys(_styles)
+    .map(key => `${key}:${_styles[key]}`)
+    .join(';');
+}
+
+export default function(storyFn) {
+  const { template, context } = storyFn();
+
+  const boundingDocument = document.createElement('div');
+  boundingDocument.innerHTML = `<div style="${stylesToString(styles.style)}">
+    <div id="component" style="${stylesToString(styles.innerStyle)}"></div>
```

Comment:
There aren't any XSS concerns for this type of project, right? In most situations you'd want to ensure `stylesToString` sanitizes anything dangerous.

Reply:
Just fixed this to do exactly what the html example does, it just assigns the styles to the Node styles so we shouldn't have to worry about XSS.